packages/oas-normalize/README.md

```diff
@@ -27,126 +27,156 @@ npm install oas-normalize
 );
 
-oas
+await oas
   .validate()
-  .then(definition => {
-    // Definition will always be JSON, and valid.
-    console.log(definition);
+  .then(() => {
+    // The API definition is valid!
   })
   .catch(err => {
-    console.log(err);
+    console.error(err);
   });
 ```
 
-### `#bundle()`
+> [!WARNING]
+> Support for Postman collections is experimental. If you've supplied a Postman collection to the library, it will **always** be converted to OpenAPI, using [`@readme/postman-to-openapi`](https://npm.im/@readme/postman-to-openapi) before doing any bundling, validating, etc.
 
-> **Note**
->
-> Because Postman collections don't support `$ref` pointers, this method will automatically upconvert a Postman collection to OpenAPI if supplied one.
+### `.load()`
+
+Load and retrive the API definition that `oas-normalize` was initialized with. Every method of `oas-normalize` utilizes this internally however if you would like to retrieve the original API _definition_ supplied (for example if all you had was a URL, a file path, or a buffer), you can use `.load()` to automatically resolve and return its contents.
+
+```ts
+const file = await oas.load();
+console.log(file);
+```
+
+### `.bundle()`
 
 Bundle up the given API definition, resolving any external `$ref` pointers in the process.
 
-```js
-await oas.bundle().then(definition => {
-  console.log(definition);
-});
+```ts
+const definition = await oas.bundle();
+console.log(definition);
 ```
 
-### `#deref()`
+### `.convert()`
+
+Convert a given API definition into an OpenAPI definition JSON object.
+
+```ts
+await oas
+  .convert()
+  .then(definition => {
+    // Definition will always be an OpenAPI JSON object, regardless if a
+    // Swagger definition, Postman collection, or even YAML was supplied.
+    console.log(definition);
+  })
+  .catch(err => {
+    console.error(err);
+  });
+```
 
-> **Note**
->
-> Because Postman collections don't support `$ref` pointers, this method will automatically upconvert a Postman collection to OpenAPI if supplied one.
+### `.deref()`
 
 Dereference the given API definition, resolving all `$ref` pointers in the process.
 
-```js
-await oas.deref().then(definition => {
-  console.log(definition);
-});
+```ts
+const definition = await oas.bundle();
+console.log(definition);
 ```
 
-### `#validate({ convertToLatest?: boolean })`
+### `.validate()`
 
-Validate and optionally convert to OpenAPI, a given API definition. This supports Swagger 2.0, OpenAPI 3.x API definitions as well as Postman 2.x collections.
+Validate a given API definition. This supports Swagger 2.0 and OpenAPI 3.x API definitions, as well as Postman 2.x collections.
 
 Please note that if you've supplied a Postman collection to the library it will **always** be converted to OpenAPI, using [@readme/postman-to-openapi](https://npm.im/@readme/postman-to-openapi), and we will only validate resulting OpenAPI definition.
```

Comment:
now that we have that callout you probably don't need this anymore:

```suggestion
```

Reply:
damn, thought i got all of them